Documentation/operations/system_requirements.rst

```diff
@@ -309,16 +309,20 @@ ICMP 8/0                 egress          ``worker-sg`` (self) health checks
 
 The following ports should also be available on each node:
 
-======================== ==========================================
+======================== ===========================================================
 Port Range / Protocol    Description
-======================== ==========================================
+======================== ===========================================================
 4240/tcp                 cluster health checks (``cilium-health``)
 4244/tcp                 Hubble server
 4245/tcp                 Hubble Relay
 6942/tcp                 operator Prometheus metrics
 9090/tcp                 cilium-agent Prometheus metrics
 9876/tcp                 cilium-agent health status API
-======================== ==========================================
+9890/tcp                 cilium-agent gops server (listening on 127.0.0.1)
+9891/tcp                 operator gops server (listening on 127.0.0.1)
+9892/tcp                 clustermesh-apiserver gops server (listening on 127.0.0.1)
```

Comment:
I don't see 9892 anywhere defined. Is this expected?

Reply:
No, that's not expected but a mistake in my manual backporting. The clustermesh-apiserver isn't part of v1.8, but I cherry-picked the doc changes from #14329 without checking. Will correct, thanks for pointing this out.